webrtc.html

```diff
@@ -4371,6 +4416,33 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       <p>An <code><a>RTCRtpSender</a></code> can be <dfn id=
       "sender-stopped">stopped</dfn>, which indicates that it will no longer
       send any media.</p>
+      <p>To <dfn>create an RTCRtpSender</dfn> with a <code>
+      <a>MediaStreamTrack</a></code>, <var>track</var>, and a list of <code>
+      <a>MediaStream</a></code> objects, <var>streams</var>, run the following
+      steps:</p>
+      <ol>
+        <li>
+          <p>Let <var>sender</var> be a new <code><a>RTCRtpSender</a></code>
+          object.</p>
+        </li>
+        <li>
+          <p>Let <var>sender.track</var> be <var>track</var>.</p>
+        </li>
+        <li>
+          <p>Let <var>sender</var> have an [[<dfn>associated
+          MediaStreams</dfn>]] internal slot, representing a list of <code>
+          <a>MediaStream</a></code> objects that the <code>
+          <a>MediaStreamTrack</a></code> object of this sender is associated
+          with.</p>
+        </li>
+        <li>
+          <p>Let <var>sender</var>'s [[<a>associated MediaStreams</a>]] slot be
```

Comment:
This should probably be *"Set* `sender`'s [[`associated MediaStreams`]] *to ..."* (you use that elsewhere).